### PR TITLE
Improve create account ux

### LINKED ISSUE #104: create account doesn't save private key
so either fix that, or tell the user that its just for testing, or, popup the private key and tell them to save it...

### PR DESCRIPTION
**Pull request by SvenDowideit**
I don't want to re-do the safe to disk, generate key, and then wallet signing interaction for 0.3.0 - that will use some of the electron wallet code that is qued up for 0.4.0

but - this at least gives the user a chance to keep the private key

closes #104

image

### CODE REVIEW

**Comment by nathanleclaire** [comment]
this is dope! we could probably have a little dialogue with save file for electron version too

**Comment by SvenDowideit** [comment]
yup, saving to file is on the 0.4.0 milestone - https://github.com/workbenchapp/solana-workbench/issues/80

File: src/renderer/components/InlinePK.tsx [hunk 1]
```diff
@@ -2,10 +2,15 @@ import { ACCOUNTS_NONE_KEY } from '../data/accounts/accountInfo';
 
 import CopyIcon from './CopyIcon';
 
-const prettifyPubkey = (pk = '') =>
-  pk !== ACCOUNTS_NONE_KEY
+const prettifyPubkey = (pk = '') => {
+  if (pk === null) {
```
Comment (nathanleclaire) [inline src/renderer/components/InlinePK.tsx:6]: it'll be `null`? not `undefined`? probably could just be `if (!pk)` ?

**Review by nathanleclaire — APPROVED** [review]

**Comment by nathanleclaire** [comment]
good fun!

**Pull request merged**